cmd/enroll.go

```diff
@@ -1,20 +1,24 @@
 package cmd
 
 import (
+	"bytes"
 	"encoding/json"
 	"errors"
 	"fmt"
+	"log"
 	"net/http"
 	"os"
 	"path/filepath"
 
+	"github.com/BurntSushi/toml"
+
 	"github.com/shopmonkeyus/eds-server/internal/util"
 	"github.com/spf13/cobra"
 )
 
 type enrollTokenData struct {
-	Token    string `json:"token"`
-	ServerID string `json:"serverId"`
+	Token    string `json:"token" toml:"server_id`
```

Comment:
```suggestion
	Token    string `json:"token" toml:"token`
```
ya gotta get that golang vscode plugin, `go vet ./...` will catch this for you